src/Chapter2.hs

```diff
@@ -336,7 +336,8 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList x y a = if x < 0 || y < 0 || x > y then []
+else take (y - x + 1) (drop x a)
```

Comment:
Does it compile with `else` starting from the beginning of a line? :thinking:

Reply:
Yes the code seems to be running fine